DVR/Tests/SessionTests.swift

```diff
@@ -26,6 +26,13 @@ class SessionTests: XCTestCase {
         } else {
             XCTFail()
         }
+
+        if let req = dataTask.currentRequest, let url = req.url {
```

Comment:
Could we just assert on pointer equality here, since `request` and `currentRequest` should match?